besca/pl/_general.py

```diff
@@ -57,6 +58,127 @@ def split_violin(tidy_data,
     #make labels 90 degrees so they are readible
     ax.tick_params(labelrotation=90,  length=6, width=2)
 
+    return(None)
+
+def box_per_ind(plotdata,y_axis,x_axis,order=None,fig_width = 4,fig_height = 3.5):
```

Comment:
It would be nice to have an example for documentation purpose. Do you have one at hand?